tests/unit/models/policy/test_patches.py

```
@@ -445,3 +447,44 @@ def permutation_kernel(x):
                 assert captured.out.count("Successfully patched") == 1
         finally:
             os.unlink(tmp_path)
+
+
+def build_sharded_3d(rank: int, world_size: int):
```

Ruff (0.14.10)

Unused function argument: `rank`

(ARG001)